projects/models.py

```diff
@@ -71,10 +71,10 @@ def get_full_image_reference(self):
 
 class Flavor(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
-    cpu_lim = models.TextField(blank=True, null=True, default="1000m")
+    cpu_lim = models.TextField(blank=True, null=True, default="2000m")
     gpu_lim = models.TextField(blank=True, null=True, default="0")
-    ephmem_lim = models.TextField(blank=True, null=True, default="200Mi")
-    mem_lim = models.TextField(blank=True, null=True, default="3Gi")
+    ephmem_lim = models.TextField(blank=True, null=True, default="1000Mi")
+    mem_lim = models.TextField(blank=True, null=True, default="4Gi")
     cpu_req = models.TextField(blank=True, null=True, default="200m")
     gpu_req = models.TextField(blank=True, null=True, default="0")
     ephmem_req = models.TextField(blank=True, null=True, default="200Mi")
```

Comment:
So I suggest adding help text to these fields, so that it'd be easier to add flavours through the admin panel

Reply:
I now added a verbose name to each of these fields, and this will be displayed in the django admin panel so hopefully it will make it easier to add flavors from there.